package.json

```diff
@@ -25,6 +25,7 @@
     "http-status-codes": "^1.3.0",
     "request": "^2.82.0",
     "source-map": "^0.5.7",
+    "strip-invalid-trailing-encoding": "^1.0.0",
```

Comment:
coolness++
maintainability--

Does 1.1.0 contain an important bug fix?

Reply:
Made the algorithm a bit more consistent. I'll update.